config/environments/production.rb

```diff
@@ -36,12 +36,15 @@
 
   # `config.assets.precompile` has moved to config/initializers/assets.rb
 
+  #run background tasks syncronously
+  config.active_job.queue_adapter = :inline
```

Comment:
for some reason, if this is not set, the deployment returns HTTP 500 for any api call...not sure why.

Reply:
Maybe @acoffman can comment on this?

Reply:
This is likely due to the default production background jobs queue requiring redis in production. Setting it to inline will cause any background tasks (delivering notifications, analytics, generating data dumps, etc) to trigger during the request/response cycle rather than being queued for later processing. 

For a smaller installation, that's probably fine, but if you notice performance issues, that could be a culprit.